docs/gettingStarted/quickStart.rst

```diff
@@ -98,7 +98,7 @@ Running CWL workflows using Toil is easy.
 
         (venv) $ toil-cwl-runner example.cwl example-job.yaml
 
-   Your output will be in ``output.txt`` ::
+   Your output will be in ``output.txt``::
```

Comment:
Sometimes you delete the space between a word and `::` and sometimes you add one.  Not a blocker, but since this is a doc fix, it's good to be consistent.

Reply:
The double colons with and without a space actually have a different meaning in ReST (see here: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks).

Essentially `word::` is shorthand for `word: ::`. So the addition and removal of a space in the examples you gave is actually equivalent to the addition and removal of a colon, which was done according to grammar rules.

Reply:
I wasn't aware of that, and I'm pretty sure I was guilty of the sections with the `word: ::` bits.  The docs have been written by a number of different people with various levels of consistency though, and I appreciate you going in and fixing a large chunk of it.